lib/gcloud/search/connection.rb

```diff
@@ -57,9 +57,34 @@ def list_indexes options = {}
         )
       end
 
+      def search index_id, query, options = {}
+        @client.execute(
+          api_method: @search.indexes.search,
+          parameters: search_request(index_id, query, options)
+        )
+      end
+
       def inspect #:nodoc:
         "#{self.class}(#{@project})"
       end
+
+      protected
+
+      def search_request index_id, query, options = {}
+        { projectId: @project,
+          indexId: index_id,
+          query: query,
+          fieldExpressions: options.delete(:expressions),
```

Comment:
The code isn't using `options` after this method is called, so I don't think we need to call `.delete` to get the values. We should be able to just do this:

```ruby
fieldExpressions: options[:expressions],
```